courses/machine_learning/deepdive2/structured/labs/1a_explore_data_babyweight.ipynb

```diff
@@ -13,7 +13,7 @@
     "\n",
```

Comment:
<p>I would prefer to describe the TODO task in markdown before the cell. Within the cell here (and for all future notebooks) we would just have "# TODO: Your code goes here"</p><p> </p><p>For example, create a markdown cell above which states:</p><p>"<strong>Lab Task:</strong> In the cells below, create a function that returns distinct value statistics from BigQuery"</p><p> </p><p>and in the cell:</p><p> </p><p>sql = # TODO: Your code goes here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/946/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>